apps/ergw_core/src/gtp_context_reg.erl

```diff
@@ -101,14 +101,9 @@ init([]) ->
 
 handle_call({await_unreg, Pid}, From, State0)
   when is_pid(Pid) ->
-    {links, Links} = process_info(self(), links),
-    case lists:member(Pid, Links) of
-	true ->
-	    State = maps:update_with(Pid, fun(V) -> [From|V] end, [From], State0),
-	    {noreply, State};
-	_ ->
-	    {reply, ok, State0}
-    end.
+    link(Pid),
+    State = maps:update_with(Pid, fun(V) -> [From|V] end, [From], State0),
+    {noreply, State}.
```

Comment:
If noreply is used, the call will be terminated by timeout. Why not return an ok?